test/memote.jl

```diff
@@ -15,20 +15,15 @@ using FBCModelTests.Memote.Metabolites
 
 @testset "Front-end" begin
     result = @testset CountTests "Testing a model that _should_ be OK" begin
-        Memote.run_tests(model_file["e_coli_core.json"], GLPK.Optimizer)
+        @test Memote.run_tests(
+            load_model(StandardModel, model_file["e_coli_core.json"]),
+            GLPK.Optimizer,
+        )
     end
 
     @test result.passes == 1783
     @test result.fails == 12
-    @test result.errs == 0
-
-    # @testset "Report can be written successfully as JSON" begin
-    #     r = Memote.generate_report(model_file["e_coli_core.json"], GLPK.Optimizer)
-    #     @test r isa Dict
-
-    #     # this should work without errors
-    #     @test JSON.json(r) isa String
-    # end
+    @test result.errs == 1 # broken counts here I think...
```

Comment:
can we document what is the error?

Reply:
(and why)

Reply:
yep, it is the energy generating test, I still need to find a way to make it non-fragile